app/src/main/java/com/fcaputo/parkingapp/utils/Constants.kt

```diff
@@ -0,0 +1,5 @@
+package com.fcaputo.parkingapp.utils
+
+object Constants {
+    const val SIZENOTSET_INT = -1
```

Comment:
this constant should be called SIZE_NOT_SET_INT to follow the standars

Reply:
Done